cli/tests/default/e2e/test_output.py

```diff
@@ -308,42 +308,6 @@ def test_omit_experiment(run_semgrep_in_tmp: RunSemgrep, snapshot):
     snapshot.assert_match(stdout, "results.out")
 
 
-@pytest.mark.kinda_slow
```

Comment:
@mjambon I've removed this test_debug_experimental_rule test.
It's not useful and worth it's annoying to udpate indeed as you wrote below.